desktop-environment.el

```diff
@@ -182,6 +192,49 @@ portion of the screen."
   (message "New volume value: %s" (desktop-environment-volume-get)))
 
 
+;;; Helper functions - keyboard backlight
+(defun desktop-environment-keyboard-backlight-percent ()
+  "Return the new keyboard backlight value as a % of maximum backlight."
+  (let ((backlight-level (desktop-environment-keyboard-backlight-get)))
+    (if (eq backlight-level 0)
+        (progn (let ((percent "0.0"))
+                 (save-match-data percent)))
```

Comment:
why do you have `save-match-data` here?

Reply:
the `progn` is useless because of the `let` which has an implicit `progn`. The `let` is useless as well:

```emacs
(if (eq backlight-level 0)
    "0.0"
  ...)
```

Reply:
As for all needless 'save-match-data' and 'let', I removed them. It is more coherent without.